_sass/_aiml.scss

```diff
@@ -63,4 +68,144 @@
     padding: 3em;
     width: calc(100% - 6em);
     position: relative;
+    flex-shrink: 0.35;
+}
+.page-container{
+    display: flex;
+    justify-content: flex-start;
+}
+.main-content{
+    width: 1000px;
+}
+
+.status-bar{
+    margin-top: 5px;
+    margin-bottom: 20px;;
+    width: 100%;
+    height: 0.5em;
+    border-radius: 10px;
+    border: hidden;
+}
+.current-lesson{
+    background: linear-gradient(to right, $ai-blue 50%, lightgray 50%);
+}
+.ai-blue-background{
+    background-color: $ai-blue;
+}
+.lightgray-background{
+    background-color: lightgray;
+}
+#desktop-sidebar{
+    width:200px;
+    margin:20px 10px;
+    margin-left: calc(50vw - 758px);
+    padding:10px 10px;
+    height: calc(100vh - 130px);
+    flex-grow: 0;
+    flex-shrink: 0;
+    border: solid;
+    border-radius: 10px;
+    overflow-y: auto;
+    overflow-x: hidden;
+    position: -webkit-sticky;
+    position: sticky;
+    top: 80px;
+    border-color: $ai-blue;
+    z-index: 1;
+    display:inline-block;
+}
+#mobile-sidebar{
+    position: fixed;
+    top: 70px;
+    left: 16px;
+    box-sizing: border-box;
+    width: 50px;
+    display: none;
+    z-index: 1;
+}
+
+// #mobile-sidebar .hamburger-icon {
+//     height: 50px;
+//     width: 50px;
+//     margin: 10px 0;
+//     display: flex;
+//     justify-content: center;
+//     flex-direction: column;
+
+//     background: white;
+//     border-radius: 10px;
+//     border: 1px solid $font-secondary-color;
+
+//     &:hover {
+//         cursor: pointer;
+
+//         .bar {
+//             margin: 3px 0px;
+//             background-color: $font-secondary-color;
+//         }
+//     }
+
+//     .bar {
+//         align-self: center;
+//         width: 50%;
+//         height: 3px;
+//         border-radius: 4px;
+//         margin: 2px 0px;
+//         transition: 0.35s cubic-bezier(0.05, 1.04, 0.72, 0.98);
+//         background-color: $font-secondary-color;
+//     }
+// }
+
+#hamburger-content{
+    display: none;
+    flex-direction: column;
+    position: fixed;
+    top: 71px;
+    left: 0px;
+    z-index: -20;
+    height: calc(100vh - 71px);
+    width: 100vw;
+    background: rgba(255, 255, 255, 1);
+    overflow-y: auto;
+    // position: fixed;
+    // top: 0;
+    // left: 0;
+    // bottom: 0;
+    // height: 100%;
+    // width: 100%;
+    // z-index: -20;
+    // transform: translate3d(0, -100%, 0);
+    // background: rgba(255, 255, 255, 0.95);
+    // display: flex;
+    // flex-direction: column;
+    // justify-content: center;
+    // transition: transform 0.35s cubic-bezier(0.05, 1.04, 0.72, 0.98);
+}
+.hamburger-item{
+    width: 350px;
+    padding: 15px 10px;
+    margin: 0px auto;
+    flex-direction: column;
+    font-size: 1.3em;
+}
```

Comment:
Make sure there's no name collision with the stuff used for the actual mobile navbar!

Reply:
I just checked. There are no name collisions.